events/views.py

```diff
@@ -1,3 +1,26 @@
-# from django.shortcuts import render
+from django.shortcuts import render, redirect
+from events.forms import EventCreationForm
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
 
-# Create your views here.
+REDIRECT_PAGE = 'welcome_page'
+
+
+@login_required(login_url=REDIRECT_PAGE)
+def create_event(request):
+
+    if request.method == 'POST':
+        print(request.POST)
+        form = EventCreationForm(request.POST, user_id=request.user)
+        if form.is_valid():
+            print("Valid form")
+            form.save()
+
+            title = form.cleaned_data.get('title')
+            messages.success(request, f'Event: {title} created!')
+            return redirect(REDIRECT_PAGE)
+
```

Comment:
Can you add an Error Message when the form is not valid.

Much like what I did in here the login-register view
https://github.com/redhat-beyond/lets-meet/blob/d0a9478c631388841015e4328b0fc04cae4e63d9/users/views.py#L30-L34

Reply:
When the form is not valid the errors can be seen in form.errors
IMHO we don't need to create error messages and just display the form.errors